packages/linter/src/migrations/update-10-4-0/update-eslint-configs-to-use-nx-presets.spec.ts

```diff
@@ -86,6 +86,12 @@ describe('Update ESLint config files to use preset configs which eslint-plugin-n
               ],
             },
           ],
+          '@nrwl/nx/no-barrel-files-import': [
+            'error',
+            {
+              disableImportFromParentAndCurrentRoot: false,
+            },
+          ],
```

Comment:
Hmm. This is test for migration for future release. Does this mean this migration will enable this rule for existing projects? For existing projects, I think enabling this rule will be a breaking change, so I don't think we should enable it by default.